api/v1/design/resource.go

```diff
@@ -0,0 +1,207 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package design
+
+import (
+	types "github.com/tektoncd/hub/api/design/types"
+	. "goa.design/goa/v3/dsl"
+)
+
+var _ = Service("resource", func() {
+	Description("The resource service provides details about all kind of resources")
+
+	HTTP(func() {
+		Path("/v1")
```

Comment:
Just a thought is it possible to define the path somewhere so that we won't have to write the whole code again instead just take the path and rest all api design take it from generalized directory (this implies mostly when api's are not removed)

Reply:
`define the path somewhere` where? 😄

Reply:
What I meant was the same thing we have done by creating a `shared`  directory, not sure will that work or not just asking

Reply:
I think for the design we can't do that we define the URL inside the service even if we share a service can't override any values

Reply:
this is fine I guess easy to maintain